content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -31,7 +31,7 @@ level of your CustomResourceDefinitions or advance your API to a new version wit
 
 The CustomResourceDefinition API supports a `versions` field that you can use to
 support multiple versions of custom resources that you have developed. Versions
-can have different schemas with a conversion webhook to convert CRs between versions.
+can have different schemas with a conversion webhook to convert custom resources between versions.
 These conversion should follow [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md) wherever applicable. Specifically
```

Comment:
These conversions or A conversion should follow the API conventions ...

Possibly rewrite, Specifically ...
See the [api change documentation](https://github.com/kubernetes/...) for a set of useful gotchas and suggestions.